### DISCUSSION TITLE
HIPAA compliance plans?

### DISCUSSION THREAD

**Discussion by SamMakesThings** [supabase/supabase#10620] [Questions]
Hi!

Saw the previous HIPAA compliance threads, but wanted to get a more recent update. Is it still in the cards, and if so, roughly when? Deciding whether I can use Supabase for a new project.

**Comment by xxfenxx** [comment]
谢谢你的来信，有有空常联系哦！

**Comment by mittcoats** [comment]
Any update on HIPAA?
